server/loadConfigs.js

```diff
@@ -4,17 +4,25 @@ const assert = require('bsert');
 const { resolve, parse } = require('path');
 
 // load main bpanel config
-function loadMainConfig(options = {}) {
-  const config = new Config('bpanel');
+/*
+ * Load up a bcfg object for a given module and set of options
+ * @params {string} - name - module name
+ * @params {object} - [options] - optional options object to inject into config
+ * @returns {Config} - returns a bcfg object
+ */
+function loadConfig(name, options = {}) {
+  assert(name && typeof name === 'string', 'Must pass a name to load config');
```

Comment:
```js
let name = undefined;
typeof name // undefined
```

Asserting name isn't undefined works, but is also captured in the `typeof` check

Reply:
Thanks. Fixing in other PR